fastq/phredQual.go

```diff
@@ -16,18 +18,31 @@ func ToQualUint8(qual []rune) []uint8 {
 	return answer
 }
 
+// ToQual will convert a slice of bytes to uint8 and subtracting a 33 offset to the values
+func ToQual(qual []byte) []uint8 {
+	var answer []uint8 = make([]uint8, len(qual))
```

Comment:
I don't think you need this copy here if you don't want it.  Since byte and uint8 should be the same type, you can just loop through "qual" subtracting 33 from each value.  However, depending on how you use this, you may be expecting []byte to be untouched or maybe overwritten, in which case you will need the copy.